packages/expo-media-library/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 
 ### 🐛 Bug fixes
 
+- [Android] Fix `getAssetsAsync` loading performance, do not load full EXIF data for images by default ([#37957](https://github.com/expo/expo/pull/37957) by [@kosmydel](https://github.com/**kosmydel**))
```

Comment:
I think it should be put inside the `breaking changes` section because of:

> As a result, some images may no longer be oriented correctly on certain Android devices when this flag is not enabled